### PR TITLE
Datavalidation module rename

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
Changed module and operator names

closes: #1592 

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/src/astro/sql/__init__.py [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
-from astro.sql.operators.data_validations.ColumnCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.check_column import (  # skipcq: PY-W2000
```
Comment (pankajastro) [inline python-sdk/src/astro/sql/__init__.py:7]: you can remove `# skipcq` see  https://github.com/astronomer/astro-sdk/pull/1489#discussion_r1071811974

File: python-sdk/src/astro/sql/__init__.py [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
     ColumnCheckOperator,
     column_check,
 )
-from astro.sql.operators.data_validations.SQLCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.check_table import (  # skipcq: PY-W2000
```
Comment (pankajastro) [inline python-sdk/src/astro/sql/__init__.py:11]: same here

File: python-sdk/src/astro/sql/__init__.py [hunk 3]
```diff
@@ -4,13 +4,13 @@
...
     SQLCheckOperator,
-    sql_check,
+    check_table,
```
Comment (feluelle) [inline python-sdk/src/astro/sql/__init__.py:13]: Why are we calling this `check_table` and in `check_column` we call it `column_check`? Let's be consistent here.

File: python-sdk/src/astro/sql/__init__.py (hunk 1 above)
Comment (utkarsharma2) [reply python-sdk/src/astro/sql/__init__.py:7]: Done

File: python-sdk/src/astro/sql/__init__.py (hunk 2 above)
Comment (utkarsharma2) [reply python-sdk/src/astro/sql/__init__.py:11]: Done.

**Comment by feluelle** [comment]
@utkarsharma2 could also update the naming of the test file, please?

**Review by feluelle — APPROVED** [review]

**Pull request merged**